addons/web/static/src/core/utils/sortable.js

```diff
@@ -66,16 +66,15 @@ export const useSortable = makeDraggableHook({
         groups: ["string", "function"],
         connectGroups: ["boolean", "function"],
     },
+    defaultParams: {
+        connectGroups: false,
+        currentGroup: null,
+        edgeScrolling: { speed: 20, threshold: 40 },
```

Comment:
With the treshold, after trying it on the runbot, I think we should increase it, perhaps even double it.

Reply:
or 60